src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.length > 0 ? (
```

Comment:
CommentList should just render list of comments. It's better to move `comments.length > 0 ?` condition to `PostInfo`